roles/openshift_storage_glusterfs/tasks/uninstall.yml

```diff
@@ -0,0 +1,10 @@
+---
+- block:
+  - include_tasks: glusterfs_config_facts.yml
+  - include_tasks: glusterfs_uninstall.yml
+    when: "'glusterfs' in groups and groups['glusterfs'] | length > 0"
```

Comment:
Move the indentation two spaces to the left. This applies it to the whole block, not the last task.

Reply:
Done